src/test/java/org/folio/edge/patron/utils/PatronOkapiClientTest.java

```diff
@@ -411,9 +493,7 @@ public void testCancelHoldSuccessfully(TestContext context) throws Exception {
             }
             async.complete();
           },
-          t -> {
-            context.fail(t);
-          });
+        context::fail);
```

Comment:
this is a good change, but the indentation seems off on these... please double check (it may just be how github is displaying it)